### ISSUE TITLE
-n 2 option not available

### ISSUE THREAD

**Issue by andywenk** [klaemo/docker-couchdb#94]
I am not sure if this is a problem or not. When following the example at https://hub.docker.com/r/klaemo/couchdb/ and do

    # Start two nodes (without proxy) exposed on port 15984 and 25984
    docker run -it -p 15984:15984 -p 25984:25984 klaemo/couchdb:2.0-dev -n 2

I get the error:

    docker: Error response from daemon: oci runtime error: exec: "-n": executable file not found in $PATH.

Docker version 1.12.1, build 6f9534c (Mac OS X)

**Comment by andywenk** [comment]
just recognised that the same problem occurs with 

    --with-admin-party-please 
    --with-haproxy

    docker: Error response from daemon: oci runtime error: exec: "--with-admin-party-please": executable file not found in $PATH.
    ....

**Comment by klaemo** [comment]
good catch! this is indeed broken. I'll fix it ASAP

**Comment by klaemo** [comment]
for the time being you can work around it by specifying ` /usr/src/couchdb/dev/run` explicitly.

```
docker run -p 5984:5984 klaemo/couchdb:2.0-dev /usr/src/couchdb/dev/run --help
```

**Comment by klaemo** [comment]
fixed in f253c65

**Comment by andywenk** [comment]
perfect - thanks!
